xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-common/src/main/resources/Main/WebHome.xml

```diff
@@ -45,11 +45,13 @@ XWiki can be used as a knowledge base (support, documentation, sales, etc.), for
 
 To make the most out of your wiki, log-in and:
 
-Use the {{displayIcon name="pencil"/}} button above to //edit// this page and start customizing your wiki to your needs.
+Use the {{displayIcon name="pencil"/}}{{html}}&lt;span class="sr-only"&gt;Edit&lt;/span&gt;{{/html}} button above to //edit// this page and start customizing your wiki to your needs.
```

Comment:
Wouldn't that be better to actually improve the `displayIcon` macro to be able to insert the alternative text?

Reply:
To be honest I'm not a big fan of inserting an `{{html}}` macro there: it makes the page content more complex.

Reply:
There is no need to use HTML, using `(% class="sr-only" %)Edit(% %)` should give the exactly same HTML but has the disadvantage that it isn't editable in the WYSIWYG editor. I agree with @surli that it might be better to integrate this as a macro parameter to make it easier to discover.

Reply:
I agree with Simon, it's very important that the home page be as simple as possible to edit/understand as it's made to be edited and modified. It also acts as example. So -1 from me too to add the html macro.

Reply:
I thought about this solution too, but then remembered `Only use icons along text alternatives. The text alternative can be visually hidden.` from https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/#HWCAG .

However, the velocity template is not API, so I guess it's okay to add an optional parameter to make it output the text alternative next to the icon itself.

I'll proceed to the change since it seems like everyone advocates for it and I have no strong opinion against it :+1:

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3123/commits/f1b8e50933acff43d95a148922b1e67f176c027e :+1: